test/system/211-dbus.bats

```diff
@@ -58,7 +58,156 @@ teardown() {
   assert [ ${#stderr_lines[@]} -eq 0 ]
 }
 
-@test "dbus: system bus" {
+@test "dbus: session bus inside Arch Linux" {
+  local expected_response
+  expected_response="$(gdbus call \
+                         --session \
+                         --dest org.freedesktop.DBus \
+                         --object-path /org/freedesktop/DBus \
+                         --method org.freedesktop.DBus.Peer.Ping)"
+
+  create_distro_container arch latest arch-toolbox-latest
+
+  run --keep-empty-lines --separate-stderr "$TOOLBOX" run --distro arch sh -c 'gdbus call \
+                                                                                 --session \
+                                                                                 --dest org.freedesktop.DBus \
+                                                                                 --object-path /org/freedesktop/DBus \
+                                                                                 --method org.freedesktop.DBus.Peer.Ping'
```

Comment:
Nitpick: we should probably wrap these lines even more aggressively, because they are beginning to become too long.

Reply:
I took the liberty to update the branch.